api/handlers/dataexplorerhandler.py

```diff
@@ -305,7 +305,39 @@ def _parse_request(self, request_type='search'):
         for f in filters:
             if f.get('terms'):
                 for k,v in f['terms'].iteritems():
-                    modified_filters.append({'terms': {k+'.raw': v}})
+                    if "null" in v:
+                        try:
+                            v.remove("null")
+                        except AttributeError:
+                            v = None
+                        null_filter = {
+                            'bool': {
+                                'should': [
```

Comment:
I thought it was needed for when multiple boxes were checked because if a must was used would that not return only docs that both have the field set to a value and not?